src/views/CashlinkCreate.vue

```diff
@@ -566,6 +597,7 @@ export default class CashlinkCreate extends Vue {
         justify-content: center;
```

Comment:
Since it already has an opacity of 0.06 on its own, and then gets even more opaque I think this change does not make any significant difference. I changed it nonetheless, as I actually like it better as a `hr` either way.